packages/video_player/video_player/lib/video_player.dart

```diff
@@ -887,12 +886,18 @@ class _VideoPlayerWithRotation extends StatelessWidget {
   final Widget child;
 
   @override
-  Widget build(BuildContext context) => rotation == 0
-      ? child
-      : Transform.rotate(
-          angle: rotation * math.pi / 180,
-          child: child,
-        );
+  Widget build(BuildContext context) {
+    if (rotation == 0) {
+      return child;
+    }
+    if (rotation % 90 != 0) {
+      throw ArgumentError('Rotation must be a multiple of 90');
```

Comment:
An argument error should be thrown at the point where it is passed as an argument, which is the constructor on line 884.

Reply:
The issue has been fixed.